features/track_bundle.feature

```diff
@@ -0,0 +1,24 @@
+Feature: track bundles
+  The ec command line can track bundles and generate tracking data files
+
+  Background:
+    Given stub registry running
+
+  Scenario:
+    Given an tekton bundle image named "acceptance/bundle:tag" containing
+      | type     | name      |
+      | Task     | task1     |
```

Comment:
These are currently not used, is that right?

Reply:
Yeah, I wanted to remove the `--type` parameter, but thought to it would be best to create a scenario for the current state, so I stopped myself before that...